src/components/Button/Button.tsx

```diff
@@ -1,12 +1,11 @@
-import { ButtonHTMLAttributes, ForwardedRef, forwardRef, ReactNode, useEffect, useState } from 'react';
+import { ButtonHTMLAttributes, ForwardedRef, forwardRef, useEffect, useState } from 'react';
 import Icon from '../Icon';
 
-export type ButtonVariants = '' | 'assertive' | 'ghost' | 'subtle';
+export type ButtonVariants = 'assertive' | 'ghost' | 'subtle';
 
 export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-  children?: ReactNode;
```

Comment:
is this intentional?

Reply:
just making sure, all good with the change on my end

Reply:
i actually think this may have been added so the autodocs would populate a description for children (see fieldset). @cblva do you recall?